man/CMakeLists.txt

```diff
@@ -13,45 +13,55 @@
-SET(MAN1_CLIENT msql2mysql.1 mysql.1 mysql_find_rows.1 mysql_waitpid.1
-		mysqldumpslow.1
-		mysqlaccess.1 mysqladmin.1 mysqlbinlog.1 mysqlcheck.1
-		mysqldump.1 mysqlimport.1 mysqlshow.1 mysqlslap.1 mytop.1
-                mysql_plugin.1 mysql_embedded.1 my_print_defaults.1
-                mysqlhotcopy.1 mysql_setpermission.1 mysql_tzinfo_to_sql.1
-                mysql_convert_table_format.1 replace.1 mysql_secure_installation.1
-		perror.1 resolve_stack_dump.1)
+SET(MAN1_CLIENT_NATIVE msql2mysql.1 perror.1 replace.1 resolve_stack_dump.1 mytop.1 my_print_defaults.1)
+SET(MAN1_CLIENT_COMPAT mysql.1 mysql_find_rows.1 mysql_waitpid.1
+                       mysqldumpslow.1
+                       mysqlaccess.1 mysqladmin.1 mysqlbinlog.1 mysqlcheck.1
+                       mysqldump.1 mysqlimport.1 mysqlshow.1 mysqlslap.1
+                       mysql_plugin.1 mysql_embedded.1
+                       mysqlhotcopy.1 mysql_setpermission.1 mysql_tzinfo_to_sql.1
+                       mysql_convert_table_format.1 mysql_secure_installation.1)
+SET(MAN1_CLIENT ${MAN1_CLIENT_NATIVE} ${MAN1_CLIENT_COMPAT})
 SET(MAN1_DEVEL mysql_config.1)
-SET(MAN1_TEST mysql-stress-test.pl.1 mysql-test-run.pl.1 mysql_client_test.1
-              mysqltest.1 mysqltest_embedded.1 mysql_client_test_embedded.1
-              my_safe_process.1)
+SET(MAN1_TEST my_safe_process.1
+              mysql-stress-test.pl.1 mysql-test-run.pl.1 mysql_client_test.1
+              mysqltest.1 mysqltest_embedded.1 mysql_client_test_embedded.1)
 IF(NOT WITHOUT_SERVER)
-  INSTALL(FILES ${MAN1_SERVER} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesServer)
-  INSTALL(FILES ${MAN8_SERVER} DESTINATION ${INSTALL_MANDIR}/man8 COMPONENT ManPagesServer)
+  INSTALL(FILES ${MAN1_SERVER_NATIVE} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesServer)
+  IF (WITH_WSREP)
+    INSTALL(FILES ${MAN1_WSREP} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesServer)
+  ENDIF()
+  INSTALL(FILES ${MAN1_SERVER_COMPAT} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ServerSymlinks)
+  INSTALL(FILES ${MAN8_SERVER} DESTINATION ${INSTALL_MANDIR}/man8 COMPONENT ServerSymlinks)
 ENDIF()
+SET(MAN1_BACKUP mbstream.1 mariabackup.1)
+INSTALL(FILES ${MAN1_BACKUP} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT backup)
 
-INSTALL(FILES ${MAN1_CLIENT} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesClient)
+INSTALL(FILES ${MAN1_CLIENT_NATIVE} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesClient)
+INSTALL(FILES ${MAN1_CLIENT_COMPAT} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ClientSymlinks)
 INSTALL(FILES ${MAN1_DEVEL} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesDevelopment)
 INSTALL(FILES ${MAN1_TEST} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT ManPagesTest)
 
+SET(MAN1_ROCKSDB mysql_ldb.1 myrocks_hotbackup.1)
+INSTALL(FILES ${MAN1_ROCKSDB} DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT rocksdb-engine)
+INSTALL(FILES aria_s3_copy.1 DESTINATION ${INSTALL_MANDIR}/man1 COMPONENT s3-engine)
```

Comment:
same for aria

Reply:
this is easy to move, but I thought there was some value in keeping all the man pages in one location.